documentation/asciidoc/computers/raspberry-pi/revision-codes.adoc

```diff
@@ -7,15 +7,15 @@ Each distinct Raspberry Pi model revision has a unique revision code. You can lo
 cat /proc/cpuinfo
 ----
 
-The last three lines show the hardware type, the revision code, and the Pi's unique serial number. For example:
+The last three lines show the hardware type, the revision code, and the Raspberry Pi's unique serial number. For example:
 
 ----
 Hardware    : BCM2835
 Revision    : a02082
 Serial      : 00000000765fc593
 ----
 
-NOTE: As of the 4.9 kernel, all Pis report `BCM2835`, even those with BCM2836, BCM2837 and BCM2711 processors. You should not use this string to detect the processor. Decode the revision code using the information below, or `cat /sys/firmware/devicetree/base/model`.
+NOTE: As of the 4.9 kernel, all Raspberry Pis report `BCM2835`, even those with BCM2836, BCM2837 and BCM2711 processors. You should not use this string to detect the processor. Decode the revision code using the information below, or `cat /sys/firmware/devicetree/base/model`.
```

Comment:
Just spotted this, "Raspberry Pi boards" perhaps, definitely not "Raspberry Pis".

Reply:
I've change this to `Raspberry Pi computers`.